manifests/example-prod/ide-modules.yaml

```diff
@@ -21,56 +21,56 @@ parameters:
         key: EksOidcArn
   - name: secrets-manager-name
     value: jh-credentials
----
-name: emrstudio-on-eks
-path: modules/beta/emrstudio-on-eks/
-parameters:
-  - name: vpc-id
-    valueFrom:
-      moduleMetadata:
-        group: optionals
-        name: networking
-        key: VpcId
-  - name: private-subnet-ids
-    valueFrom:
-      moduleMetadata:
-        group: optionals
-        name: networking
-        key: PrivateSubnetIds
-  - name: eks-cluster-admin-role-arn
-    valueFrom:
-      moduleMetadata:
-        group: core
-        name: eks
-        key: EksClusterAdminRoleArn
-  - name: eks-cluster-name
-    valueFrom:
-      moduleMetadata:
-        group: core
-        name: eks
-        key: EksClusterName
-  - name: eks-oidc-arn
-    valueFrom:
-      moduleMetadata:
-        group: core
-        name: eks
-        key: EksOidcArn
-  - name: eks-openid-issuer
-    valueFrom:
-      moduleMetadata:
-        group: core
-        name: eks
-        key: EksClusterOpenIdConnectIssuer
-  - name: artifact-bucket-name
-    valueFrom:
-      moduleMetadata:
-        group: optionals
-        name: datalake-buckets
-        key: ArtifactsBucketName
-  - name: sso-username
-    value: replace-me
-  - name: emr-eks-namespace
-    value: emr-studio
+# ---
```

Comment:
we probably need to drop this beta module from example-prod completely, not just comment it.

Reply:
Thanks for catching it. As discussed offline earlier, i have removed the entries of emrstudio in the release/0.1.0 but i missed addressing them in main.